pkg/apis/applicationnetworking/v1alpha1/accesslogpolicy_types.go

```diff
@@ -81,6 +81,14 @@ func (p *AccessLogPolicy) GetTargetRef() *v1alpha2.PolicyTargetReference {
 	return p.Spec.TargetRef
 }
 
+func (p *AccessLogPolicy) GetStatusConditions() []metav1.Condition {
+	return p.Status.Conditions
+}
+
+func (p *AccessLogPolicy) SetStatusConditions(conditions []metav1.Condition) {
+	p.Status.Conditions = conditions
+}
```

Comment:
What are these for?

Reply:
I plan to in the next PR to change

```
func (r *accessLogPolicyReconciler) updateAccessLogPolicyStatus(
	ctx context.Context,
	alp *anv1alpha1.AccessLogPolicy,
	reason gwv1alpha2.PolicyConditionReason,
)
 ```

to a more general function:
```
func UpdatePolicyStatus(
	ctx context.Context,
	k8sClient client.Client,
	policy core.Policy,
	reason gwv1alpha2.PolicyConditionReason,
) error {
```

Reply:
That would be fantastic, thank you!